osu.Game.Tests/Visual/Online/TestSceneUserProfileOverlay.cs

```diff
@@ -90,7 +90,9 @@ public void TestLoading()
             {
                 new APIUserGroup { Colour = "#EB47D0", ShortName = "DEV", Name = "Developers" },
                 new APIUserGroup { Colour = "#A347EB", ShortName = "BN", Name = "Beatmap Nominators", Playmodes = new[] { "mania" } },
-                new APIUserGroup { Colour = "#A347EB", ShortName = "BN", Name = "Beatmap Nominators", Playmodes = new[] { "osu", "taiko" } }
+                new APIUserGroup { Colour = "#A347EB", ShortName = "BN", Name = "Beatmap Nominators", Playmodes = new[] { "osu", "taiko" } },
```

Comment:
This element needs proper tests at this point. You can pretty much just take https://github.com/ppy/osu/pull/22280/files#diff-76fbb094d3f6a30d2f95a68e009bf41f57bd09daeb1ef4e76ff23ce94f33835f and remove the combined part

Reply:
Pushed a commit that does this, mark as resovled if its good enough